src/js/actions/index.js

```diff
@@ -0,0 +1 @@
+//Actions here
```

Comment:
This file looks like it might be from work on #163, probably don't need it here.